scripts/restore-sync/template.job.yaml

```diff
@@ -10,13 +10,13 @@ spec:
       restartPolicy: Never
       containers:
         - name: rclone
-          image: elastisys/rclone-sync:1.3.0
+          image: ghcr.io/elastisys/rclone-sync:1.63.0
```

Comment:
If we change this tag to following the version of rclone, then I'd like to pin that version in the Dockerfile.
So to change the

```bash
RUN curl -O https://downloads.rclone.org/rclone-current-linux-amd64.zip && \
```

into

```bash
ENV RCLONE_VERSION="v1.63.0"
RUN curl -O "https://downloads.rclone.org/${RCLONE_VERSION}/rclone-${RCLONE_VERSION}-linux-amd64.zip" && \
```